File: src/main/java/com/qtech/bubbles/util/RegistryUtils.java
package com.qtech.bubbles.util;

import com.qtech.bubbles.registry.object.ObjectHolder;
import org.jetbrains.annotations.Nullable;

public class RegistryUtils {
    @Nullable
    public static ObjectHolder getObjectHolder(Class<?> clazz) {
        return clazz.getDeclaredAnnotation(ObjectHolder.class);
    }
}

File: src/main/kotlin/com/qtech/bubbles/util/RegistryUtils.kt
package com.qtech.bubbles.util

import com.qtech.bubbles.registry.`object`.ObjectHolder

object RegistryUtils {
    fun getObjectHolder(clazz: Class<*>): ObjectHolder? {
        return clazz.getDeclaredAnnotation(ObjectHolder::class.java)
    }
}